pkgs/development/python-modules/jsons/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy36
+, attrs
+, dataclasses
+, typish
+, tzdata
+}:
+
+buildPythonPackage rec {
+  pname = "jsons";
+  version = "1.6.3";
+
+  src = fetchFromGitHub {
+    owner = "ramonhagenaars";
+    repo = "jsons";
+    rev = "9abbf3a3bd32435ac74bc98c3554ad3c71086036";
```

Comment:
Why by commit id?

Reply:
(I'll expand on that..)  doing this by commit id isn't very transparent (readers have to trust that the above isn't some random commit that's not even in one of the upstream author's branches) and it also won't work with auto-updaters like r-ryantm.

Reply:
Changed to use the `v1.6.3` tag

Reply:
We tend to do

```
  rev = "v${version}";
```

in cases like this because it makes things simpler for auto-updaters.

Reply:
Updated to use this